### PR TITLE
ip/ffmpeg: decode multiple frames from one packet

### PR DESCRIPTION
**Pull request by Ashymad**
See here https://github.com/cmus/cmus/issues/812#issuecomment-495201322 for a breakdown of the problem.
This is mostly a hack, but fixes the problem as far as i know. APE files play without a problem (fixes https://github.com/cmus/cmus/issues/812). It should also fix https://github.com/cmus/cmus/issues/895, but I don't have any WMA files to test it.

### CODE REVIEW

File: ip/ffmpeg.c [hunk 1]
```diff
@@ -37,7 +37,7 @@
 #endif
 
 #ifndef AVCODEC_MAX_AUDIO_FRAME_SIZE
-#define AVCODEC_MAX_AUDIO_FRAME_SIZE 192000
+#define AVCODEC_MAX_AUDIO_FRAME_SIZE 1920000
```
Comment (Ashymad) [inline ip/ffmpeg.c]: To fit the additional frames I had to increase the size of this constant. I've increased it tenfold, but this obviously makes little sense. I don't have an idea how to approach this without changing too much code.

**Comment by nefthy** [comment]
Thanks for your effort, but I don't think its wise to merge this PR as it is obviously a hacky workaround. A proper fix should adress the issue as described in the documentation linked in the mentioned comment.

**Comment by Ashymad** [comment]
A proper fix would probably entail rewriting quite a bit of the plugin. Although this is hacky, it mostly adheres to documentation (the only real problems are the constant buffer size and lack of proper error handling) and only touches the code, that would have to be rewritten anyway (there is no error handling in the original code as well). I think merging this for now doesn't significantly decrease the code quality, whilst giving us a bit more time with a working plugin until someone willing to really fix it steps up.

**Comment by nefthy** [comment]
Closing due to inactivity. Please feel free to reopen or create a new one.

**Pull request closed without merging**